packages/web/src/common/store/account/sagas.ts

```diff
@@ -0,0 +1,50 @@
+import { accountActions, accountSelectors } from '@audius/common/store'
```

Comment:
is the `waitForRead` the only reason this can't be in true common?

do we even need that helper? could we depend on an account selector instead?

Reply:
`retrieveCollections` is what blocks this from going to common. Which would mean moving all the cache stuff over there as well.
This could/should go away and be replaced with some simple hooks once the tan-query stuff is done.

Reply:
cool